ledger/participant-state/kvutils/tools/src/main/scala/com/daml/ledger/participant/state/kvutils/tools/CommitStrategySupport.scala

```diff
@@ -23,5 +23,9 @@ trait CommitStrategySupport[LogResult] {
   def createComponents()(implicit executionContext: ExecutionContext)
     : (DamlLedgerStateReader, CommitStrategy[LogResult], QueryableWriteSet)
 
-  def explainMismatchingValue(key: Key, expectedValue: Value, actualValue: Value): String
+  /**
+    * Determines if there's an actual difference and tries to explain it in case there is.
+    * @return  None in case no difference should be signaled to the user; otherwise a message explaining the difference
```

Comment:
```suggestion
    * Determines if there's an actual difference and tries to explain it.
    *
    * @return  None in case no difference should be signaled to the user; otherwise a message explaining the difference
```